tools/tpm2_import.c

```diff
@@ -288,6 +293,13 @@ static bool create_import_key_public_data_and_name(void) {
 
     IMPORT_KEY_SYM_PUBLIC_AREA(ctx.import_key_public)
 
+    if (ctx.flags.A) {
+        ctx.import_key_public.t.publicArea.objectAttributes.val = ctx.objectAttributes;
+    }
+
+    tpm2_tool_output("ObjectAttribute: 0x%08X\n",
+                     ctx.import_key_public.t.publicArea.objectAttributes.val);
+
```

Comment:
I think this should only be printed if ctx.objectAttributes was assigned a value.

Reply:
This also refers to tpm2_create and tpm2_createprimary.